src/test/end-to-end/tests/login-test.ts

```diff
@@ -0,0 +1,46 @@
+Feature('Home page');
+
+Scenario('I as a system admin should be able to sign-in with the valid credentials', async ({I}) => {
+  I.loginAsSystemAdmin();
+  I.see('System Admin Dashboard');
+  I.logout();
```

Comment:
After logout, think we come to a page which says "You have been logout". Can we do a "I.see" after logout?

Reply:
I have included that in the function, I think you didn't notice
logout: function () {
      this.click('Sign out');
      this.see('You have been signed out');
    },